backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5774790_AddDatesInOrder.sql

```diff
@@ -0,0 +1,167 @@
+;
+
+-- 2025-10-27T16:32:44.643Z
+/* DDL */  select update_Column_Translation_From_AD_Element(578690)
+;
+
+-- 2025-10-27T16:32:46.555Z
+/* DDL */ SELECT public.db_alter_table('C_Order','ALTER TABLE public.C_Order ADD COLUMN InvoiceDate TIMESTAMP WITHOUT TIME ZONE')
+;
+
+-- Field: Bestellung_OLD(181,D) -> Bestellung(294,D) -> ETA
+-- Column: C_Order.ETA
+-- 2025-10-28T08:40:32.265Z
+INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,ColumnDisplayLength,Created,CreatedBy,Description,DisplayLength,EntityType,FacetFilterSeqNo,IncludedTabHeight,IsActive,IsDisplayed,IsDisplayedGrid,IsEncrypted,IsFieldOnly,IsHeading,IsHideGridColumnIfEmpty,IsOverrideFilterDefaultValue,IsReadOnly,IsSameLine,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,SeqNoGrid,SortNo,SpanX,SpanY,Updated,UpdatedBy) VALUES (0,591441,755107,0,294,0,TO_TIMESTAMP('2025-10-28 08:40:32.044000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'Voraussichtliches Ankunftsdatum',0,'D',0,0,'Y','Y','Y','N','N','N','N','N','N','N',0,'ETA',0,0,230,0,1,1,TO_TIMESTAMP('2025-10-28 08:40:32.044000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
+
+-- 2025-10-28T08:40:32.280Z
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Field_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Field t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Field_ID=755107 AND NOT EXISTS (SELECT 1 FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 2025-10-28T08:40:32.290Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(584067)
+;
+
+-- 2025-10-28T08:40:32.315Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=755107
+;
+
+-- 2025-10-28T08:40:32.328Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(755107)
+;
+
+-- Field: Bestellung_OLD(181,D) -> Bestellung(294,D) -> B/L Datum
+-- Column: C_Order.BLDate
+-- 2025-10-28T08:40:41.682Z
+INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,ColumnDisplayLength,Created,CreatedBy,Description,DisplayLength,EntityType,FacetFilterSeqNo,IncludedTabHeight,IsActive,IsDisplayed,IsDisplayedGrid,IsEncrypted,IsFieldOnly,IsHeading,IsHideGridColumnIfEmpty,IsOverrideFilterDefaultValue,IsReadOnly,IsSameLine,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,SeqNoGrid,SortNo,SpanX,SpanY,Updated,UpdatedBy) VALUES (0,591442,755108,0,294,0,TO_TIMESTAMP('2025-10-28 08:40:41.483000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'',0,'D',0,0,'Y','Y','Y','N','N','N','N','N','N','N',0,'B/L Datum',0,0,240,0,1,1,TO_TIMESTAMP('2025-10-28 08:40:41.483000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
```

Comment:
there is no field description that explains what BL stands for, what it is used for, why it is read-only in the purchase-order, the it is set instead set in the transportation order and then the value used to compute Order Payment Schedules.
We need this description
- in EN and DE
- using the acutal correct terms (window/tab/field names)
- created with ❤️